Lib/test/test_unpack.py

```diff
@@ -167,5 +220,49 @@ def test_extended_oparg_not_ignored(self):
             y = unpack_400(range(400))
             self.assertEqual(y, 399)
 
+    def test_exception_context_when_len_fails(self):
+        """When `__len__()` raises an Exception, preserve exception context"""
```

Comment:
```suggestion
        """Preserve exception context when __len__() raises an Exception (but not a BaseException)."""
```

While I love Star Wars, Obi-Wan is usually better understood than Master Yoda.

Reply:
The full docstring would be “Test that exception context is preserved when `__len__()` raises an Exception”. IMO, the version before this change is OK as a shortened version.

Reply:
Fine by me.